families/index.html

```diff
@@ -33,4 +36,4 @@ <h3>Here are the loving families with their fuzzy bunnies</h3>
     <script type="module" src="./families.js"></script>
 </body>
 
-</html>
+</html>
```

Comment:
make sure to have an extra line of whitespace at the end to avoid these warnings. Message me if you want some tips on setting up eslint and prettier to handle this for you!